functest/get_balance_test.go

```diff
@@ -19,19 +19,23 @@
 package functest
 
 import (
+	"math/big"
 	"testing"
 
 	"github.com/insolar/insolar/testutils"
+
 	"github.com/stretchr/testify/require"
 )
 
 func TestGetBalance(t *testing.T) {
 	firstMember := createMember(t, "Member1")
 	firstBalance := getBalanceNoErr(t, firstMember, firstMember.ref)
-	require.Equal(t, 1000*1000*1000, firstBalance)
+	r := big.NewInt(1000000000)
+	require.Equal(t, r, firstBalance)
 }
 
 func TestGetBalanceWrongRef(t *testing.T) {
 	_, err := getBalance(&root, testutils.RandomRef().String())
-	require.Contains(t, err.Error(), "[ getBalanceCall ] : [ GetDelegate ] on calling main API")
+	require.NotNil(t, err)
+	require.Contains(t, err.Error(), "lifeline not found", "reference does not exist")
```

Comment:
`Contains(t TestingT, s interface{}, contains interface{}, msgAndArgs ...interface{})` - last param is the message, which will be displayed if check failed. In this case it will be:
`Error:      	"some not expected error" does not contain "lifeline not found"`
        	`Messages:   	reference does not exist`
It seems a little off.